packages/relay/src/lib/services/clientService/index.ts

```diff
@@ -0,0 +1,276 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import dotenv from 'dotenv';
+import findConfig from 'find-config';
+import { AccountId, Client, PrivateKey } from '@hashgraph/sdk';
+import { Logger } from 'pino';
+import { Registry, Counter } from 'prom-client';
+import { SDKClient } from '../../clients/sdkClient';
+
+export default class ClientService {
+  private transactionCount: number;
+  private errorCount: number;
+  private resetDuration: number;
+  private shouldReset: boolean;
+
+  private isEnabled: boolean;
+  private isTimeResetDisabled: boolean;
+
+  private clientMain: Client;
+
+  /**
+   * The SDK Client use for connecting to both the consensus nodes and mirror node. The account
+   * associated with this client will pay for all operations on the main network.
+   *
+   * @private
+   */
+  private client: SDKClient;
+
+  /**
+   * The logger used for logging all output from this class.
+   * @private
+   */
+  private logger: Logger;
+  private readonly register: Registry;
+  private clientResetCounter: Counter;
+
+  private chainIds = {
```

Comment:
Move to constants or make public static

Reply:
and  also  refactor `relay.eth`  to  use the constant.
https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/src/lib/relay.ts#L35

Reply:
Added to constants.